team-policy.md

```diff
@@ -0,0 +1,138 @@
+
+The agenda backlog and minutes document is structured according to the following template:
+
+```md
+# T-style Minutes
+
+Meeting Link: https://meet.jit.si/t-style-style-team-task-force
+
+## Action Items
+
+### Pending
+
+* owner: bullet point list of items that are in progress and assigned to a specific person
+
+### Completed
+
+- [ ] owner: check list of items that are completed or assumed to be complete
+- [x] owner: items are checked off once they've been reviewed in a meeting, confirmed complete, and given any relevant final status updates.
+- [x] owner: after a meeting the items that were checked off are moved into the `#### Completed Action Items` section of the meeting they were reviewed in by the secretary
+
+## Backlog
+
+* Bullet point list of items that have not been started or assigned yet
+
+## Next Meeting Date
+
+### Attendance
+
+### Agenda
+
+* (inform) bullet point list of proposed agenda items (labeled either inform, explore, or decide)
+* Meeting Check-out
+
+### Minutes
+
+#### Individual Agenda Items
+
+Notes related to the given agenda item
+
+#### Completed Action Items
+
+## Previous Meeting Dates <repeats>
+
+```
+### Meeting Check-out
+
+The style team wraps up the content part of our meetings 5-10 minutes before team members have to leave to make room for regular meeting evaluations. Meeting evaluations are an integral part of every meeting. We end the meeting with one or two rounds on:
+
+* “What worked well in the meeting?”
+* “What could be improved in future meetings?”
+* “Is there anything you are carrying out of the meeting that you’d like to get off your chest now?”
+
+Meeting evaluations are an opportunity to learn from our meetings. We can either talk about content, process, or interpersonal dynamics. We utilize meeting evaluations to help ourselves inhabit a growth mindset. Our goal is to have meetings which are well-run, refreshing, connecting, and energizing. We achieve this goal by giving space for people to speak up about both the positive and negative aspects of how we're working together so that we can prioritize continuous improvement and positive connections.
+
+## Style Guide Evolution
+
+The Rust style guide will generally match the latest version of the Rust style; the style team does not plan to maintain multiple branches of the style guide for different editions, in part because formatting for new constructs will apply to any edition supporting those constructs.
+
+Whenever possible, style decisions should be made before a new construct is stabilized. However, style decisions shall not be considered a blocker for stabilization.
```

Comment:
What would you think about moving this section somewhere higher in the document, at least before the meeting minutes template?

I ask because I feel like it's one of the more critical and impactful decisions we've made and worry that it might get a little lost at the bottom following the meeting template (especially for those reading the non-rendered markdown)